R/documentation.R

```diff
@@ -192,6 +192,53 @@ NULL
 #' @name Button
 NULL
 
+#' FileUploadButton
+#'
+#' @description
+#' A Safari-compatible file upload button that combines Fluent UI styling with cross-browser file selection functionality. This component solves the issue where Safari blocks programmatic file input clicks by using React-based file input handling.
```

Comment:
Style: this wording seems oddly focused around Safari. I don't think the emphasis makes sense as it is only relevant in the context of #241.

```suggestion
#' A Safari-compatible file upload button that combines Fluent UI styling with cross-browser file selection functionality. This component handles Safari blocking programmatic file input clicks by using React-based file input handling.
```